Formula/gdk-pixbuf.rb

```diff
@@ -38,6 +38,9 @@ def gdk_module_ver
   end
 
   def install
+    # fix libtool versions
+    # https://bugzilla.gnome.org/show_bug.cgi?id=776892
```

Comment:
Should we mention explicitly that this needs to be removed for 2.38.0 and later?